examples/structs/structs.go

```diff
@@ -12,6 +12,16 @@ type person struct {
 	age  int
 }
 
+// A de facto constructor of type `person`.
+func NewPerson(name string) *person {
+
+	// You can safely return a pointer to local variable
+	// as a local variable will survive the scope of the function.
```

Comment:
// You can safely return a pointer to a local variable. Go will
    // allocate such variables on the heap.